tests/group/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+cmake_minimum_required (VERSION 3.0) # The minimum version of CMake necessary to build this project
+project (group) # The name of our project
+
+declare_trisycl_test(TARGET group TEST_REGEX
+" 10
+ 2
+ 0
+ 5
+5 
```

Comment:
Thank you for the explanation.
Most of these tests grew organically as a way to explain what was happening in SYCL too.
But at the end they are not correct device-compatible or deterministic tests, so I agree with you that they should be rewritten without any I/O.